exchange.htm

```diff
@@ -2,64 +2,4 @@
 title: "Gridcoin - Exchanges"
 description: "Where to buy Gridcoin"
 ---
-<section class="acquire-details grc-bgcolor pt-5 mt-5">
-    <div class="container">
-        <div class="row text-center">
-            <div class="col-12 col-sm-12 col-md-12 col-lg-12">
-                <h3>How to buy Gridcoin?</h3>
-                <p>On most exchanges you need to first <a href="https://www.bitcoin.com/buy-bitcoin">buy Bitcoin</a> in order to buy Gridcoin.</p>
-            </div>
-        </div>
-        <div class="row text-center">
-            <!--
-            <div class="col-12 col-sm-4 col-md-4 col-lg-4">
-                <h3>Decentralized exchanges</h3>
-                <ul>
-                    
-                    <li>
-                        <a href="https://market.rudex.org/#/asset/RUDEX.GRC">
-                            <img src="../assets/img/Exchanges/RUDEX.png" alt="RUDEX" />
-                        </a>
-                    </li>
-                    <li>
-                        <a href="https://openledger.io/#/market/OPEN.GRC_OPEN.BTC">
-                            <img src="../assets/img/Exchanges/OL.png" alt="OpenLedger" />
-                        </a>
-                    </li>
-                </ul>
-            </div>
-            -->
-            <div class="col-12 col-sm-6 col-md-6 col-lg-6">
-                <h3>Centralized exchanges</h3>
-                <ul>
-                    <li>
-                        <a href="{{exchange.link}}">
-                            <img width="300" src="{{exchange.image}}" alt="{{exchange.name}} logo">
-                        </a>
-                    </li>
-                </ul>
-            </div>
-
-            <div class="col-12 col-sm-6 col-md-6 col-lg-6">
-                <h3>Free/Starter Gridcoin</h3>
```

Comment:
this part probably shouldn't be deleted

Reply:
as far as I can see the contents of the acquire_details template and this are mostly identical

Reply:
Exchange.htm as of this PR
![image](https://user-images.githubusercontent.com/22572406/143727100-af9d5e44-dc4e-4430-b919-d8f588e747b3.png)

Old exchange.htm
![image](https://user-images.githubusercontent.com/22572406/143727150-a392cf44-aa4e-496d-be58-22efa6ac7aca.png)

Reply:
what happened to flyp.me?

Reply:
idk why it wasn't there before

Reply:
@barton2526 sorry for making it confusing but top is this PR and bottom is before.